### ISSUE TITLE
windows download: fix wordings

### ISSUE THREAD

**Issue by haraldschilly** [sagemath/website#174]
windows-download says

> Below are the VirtualBox images

but there are actually the installers.

should we also add a page for the (previous) OVA images? or are they deprecated and it's better to use the windows installer?

**Comment by slel** [comment]
Yes, the wording should be modified.

Regarding the ova images, I don't think they are, or should be,
deprecated. They can be useful for a number of platforms
not supported by SageMath in any other way.

@vbraun, is there a way you could resume producing these
ova images? Users occasionally ask for an updated version
on the support channels.

**Comment by kcrisman** [comment]
Agreed that the ova should not be completely deprecated, but also that we are not producing them currently.
